docs/guide/docs/channels/web.md

```diff
@@ -72,51 +71,93 @@ Here is some sample code for adding the event listeners to your custom elements:
+
+```js
+      document.getElementById('hide-bp').addEventListener('click', function() {
+        window.botpressWebChat.sendEvent({ type: 'hide' })
+      })
+```
+### Toggle
+The following function includes a ternary operator that toggles the chat to either be hidden or shown when you click the `toggle-bp` element. 
+
+```js
+      document.getElementById('toggle-bp').addEventListener('click', function() {
+        window.botpressWebChat.sendEvent({ type: webchatOpen ? 'hide' : 'show' })
+      })
+```
+
+### Message
+The following function lets you programmatically send a message to the user when you click the `send-message-bp` element.
 
-It may be useful to fetch the current visitor ID to either save it in your database or to update some attributes in the Botpress DB.
+```js
+      document.getElementById('send-message-bp').addEventListener('click', function() {
+        window.botpressWebChat.sendEvent({ type: 'message', text: 'Hello!' })
+      })
+```
+
+### Obtaining visitor's User ID
 
-Since the webchat is running in an iframe, communication between frames is done by posting messages.
-The chat will dispatch an event when the user id is set, which you can listen for on your own page.
+The most widely used object property under events is `userId`. It fetches the current visitor ID to either save it in your database or update some Botpress DB attributes.
+
+Since the webchat is running in an iframe, posting messages can achieve communication between frames. The chat will dispatch an event when a user id is set. You can listen for this event on your page.
 
 ```js
 window.addEventListener('message', message => {
   if (message.data.userId) {
     console.log(`The User ID is ` + message.data.userId)
+    //Add code to use the user ID here
   }
 })
 ```
 
-## Configure at runtime
+## Runtime configurations
+The method `window.botpressWebChat.configure` allows you to change the chat's configuration during a conversation without having to reload the page. 
 
-The method `window.botpressWebChat.configure` allows you to change the configuration of the chat during a conversation without having to reload the page.
+The example below shows how you can change the chatbot's language to French when you click `change-lang-bp` on your website. 
 
-## Advanced Customization
+```js
+      document.getElementById('change-lang-bp').addEventListener('click', function() {
+        lastConfig.locale = 'fr'
+        window.botpressWebChat.configure(lastConfig)
+      })
+```
 
-Every message sent by the bot to a user consist of a `payload`. That payload has a `type` property, that tells the webchat how the other information included on that payload should be rendered on screen.
+## Advanced Customization
+All messages which the bot sends to a user consist of a `payload`. That payload has a `type` property that tells the webchat how the other information included on that payload should be rendered on screen.
```

Comment:
Should this section have a title? Sending a Custom Payload to a User? or something else more appropriate maybe?